radicle/src/identity/doc.rs

```diff
@@ -122,6 +145,11 @@ impl<V> Doc<V> {
         repo.reference_oid(remote, &git::refs::storage::IDENTITY_BRANCH)
             .map_err(DocError::from)
     }
+
+    pub fn blob_at<R: ReadRepository>(commit: Oid, repo: &R) -> Result<git2::Blob, DocError> {
```

Comment:
I'm thinking it might be better to have these as helper functions rather than on `Doc` since they don't use `self` or return `Self`. It means we don't need to use it by specifying a `V` for `Doc` on calling the functions.

Reply:
Interesting, yes we can look into that :+1: